### ISSUE TITLE
Better scrollbars

### ISSUE THREAD

**Issue by dsr432** [minbrowser/min#1204]
The scrollbars on Min right now look awful on windows (I'm not sure about Mac), but I think a big selling point for Min is the quality UI, and that just takes away from it.  I'm not sure if this is possible to change as I haven't looked through the code base or electron that much, but if it is that would be amazing.  
![image](https://user-images.githubusercontent.com/66963044/90547318-543f6080-e159-11ea-88ca-c223ff162d1e.png)

**Comment by PalmerAL** [comment]
Yeah, this is mostly dependent on Chromium. It looks like they're working on a project right now to fix this:
https://techdows.com/2020/06/chrome-scrollbar-dark-mode.html
https://bugs.chromium.org/p/chromium/issues/detail?id=1095875

so it probably makes the most sense to wait for that.

In the meantime, it's possible to [style scrollbars with CSS](https://css-tricks.com/almanac/properties/s/scrollbar/), so I think what you could do is create a [userscript](https://github.com/minbrowser/min/wiki/userscripts) that runs on websites with a dark theme, and adds some CSS to the page to make the scrollbar dark.

On macOS, the auto-hiding scrollbars for trackpads adjust to dark mode, but the standard ones for mice don't.

**Comment by dsr432** [comment]
Perhaps I should have been more clear... the color I don't think matters as much as the actual appearence.  Perhaps I'm wrong and this depends on the website, but I'm fairly certain (especially because electron itself runs in the browser, that it is possible to make custum scrollbars for Min? 
![image](https://user-images.githubusercontent.com/66963044/90562481-3df0cf00-e170-11ea-8a1b-1db42e97bf83.png)

Again I'm not experienced in electron or Min in general so correct me if I'm wrong but I think that this is possible to achieve and would look much better than the current blocky scrollbar that we have.

**Comment by PalmerAL** [comment]
Sorry, I misunderstood. I think that would be doable using the same method I mentioned above (custom CSS); I expect that it would work on most websites, although it might break if the website assumes the scrollbar is the standard width.

The bigger issue with doing that by default is that a thinner scrollbar is harder to drag with the mouse; macOS solves that by only making the scrollbar thinner if you're using a trackpad, but I don't think Windows has anything similar.

**Comment by ivancizik** [comment]
Hi, i also don't like the default one so i created the script. The script is available [here](https://github.com/ivancizik/min-css-scrollbar-userscript).

For CSS i just used a simple example from [W3Schools](https://www.w3schools.com/howto/tryit.asp?filename=tryhow_css_custom_scrollbar)

BR,
Ivan

**Comment by Syndamia** [comment]
@ivancizik have you considered adding the script to the [examples in the userscripts wiki page](https://github.com/minbrowser/min/wiki/userscripts#examples)?

**Comment by ivancizik** [comment]
@Syndamia done. :)
